src/diffpy/srmise/dataclusters.py

```diff
@@ -82,7 +82,8 @@ def __eq__(self, other):
         )
 
     def clear(self):
-        """Clear all members, including user data."""
+        """Remove all the points from the cluster list,
```

Comment:
please write docstrings using numpy standard.

The first line is a short, high level description of what the method does, then a blank line, then more details, then any parameters and what is returned, then usage examples etc.  We often don't add usage examples or even long descriptions, so use your judgement.

The docstring is very important.  If you can't write a good docstring it means you don't understand what a method is for.  If you don't understand that you can't write good tests, and if you can't do that you can't write a good function.  So spend some effort on getting the docstring right first before doing other things, then return to the docstring later if your understanding changed while writing the tests.